src/content/1.7/modules/sample_modules/view-order-hooks-demo.md

```diff
@@ -0,0 +1,721 @@
+
+ {{< figure src="../img/view-order-hooks-demo/ps-view-order-page-hooks.jpg" title="Order page hooks locations" >}}
+ 
+ ```php
+ 
+    private function registerHooks(Module $module): bool
+    {
+        // All hooks in the order view page.
+        $hooks = [
+            'displayBackOfficeOrderActions',
+            'displayAdminOrderTabContent',
+            'displayAdminOrderTabLink',
+            'displayAdminOrderMain',
+            'displayAdminOrderSide',
+            'displayAdminOrder',
+            'displayAdminOrderTop',
+            'actionGetAdminOrderButtons',
+        ];
+
+        return (bool) $module->registerHook($hooks);
+    }
+ ```
+
+Let's start from the first one - `displayBackOfficeOrderActions` and create a demo for it. 
+
+### Create User Signature card below the Customer card.
+
+Lets create `Installer` class inside `/demovieworderhooks/src/Install` forlder structure
+ responsible for creating `signature` table in the database:
+
+```php
+<?php
+
+declare(strict_types=1);
+
+namespace PrestaShop\Module\DemoViewOrderHooks\Install;
+
+use Db;
+use Module;
+
+/**
+ * Class responsible for modifications needed during installation/uninstallation of the module.
+ */
+class Installer
+{
+    /**
+     * @var FixturesInstaller
+     */
+    private $fixturesInstaller;
+
+    public function __construct(FixturesInstaller $fixturesInstaller)
+    {
+        $this->fixturesInstaller = $fixturesInstaller;
+    }
+
+    /**
+     * Module's installation entry point.
+     *
+     * @param Module $module
+     *
+     * @return bool
+     */
+    public function install(Module $module): bool
+    {
+        if (!$this->registerHooks($module)) {
+            return false;
+        }
+
+        if (!$this->installDatabase()) {
+            return false;
+        }
+
+        $this->fixturesInstaller->install();
+
+        return true;
+    }
+
+    /**
+     * A helper that executes multiple database queries.
+     *
+     * @param array $queries
+     *
+     * @return bool
+     */
+    private function executeQueries(array $queries): bool
+    {
+        foreach ($queries as $query) {
+            if (!Db::getInstance()->execute($query)) {
+                return false;
+            }
+        }
+
+        return true;
+    }
+
+    /**
+     * Install the database modifications required for this module.
+     *
+     * @return bool
+     */
+    private function installDatabase(): bool
+    {
+        $queries = [
+            'CREATE TABLE IF NOT EXISTS `'._DB_PREFIX_.'signature` (
+              `id_signature` int(11) NOT NULL AUTO_INCREMENT,
+              `id_order` int(11) NOT NULL,
+              `filename` varchar(64) NOT NULL,
+              PRIMARY KEY (`id_signature`),
+              UNIQUE KEY (`id_order`)
+            ) ENGINE='._MYSQL_ENGINE_.' DEFAULT CHARSET=utf8;'
+        return $this->executeQueries($queries);
+    }
+}
+
+```
+
+Let's create `composer.json` in the root of the module, for example:
```

Comment:
Please add a small link to Composer and explain why we use it (it does the autoload for us, no need for require_once everywhere)